beacon-chain/chaintest/state-replay/main.go

```diff
@@ -0,0 +1,151 @@
+package main
+
+import (
+	"context"
+	"flag"
```

Comment:
please organize imports into two groups where the top group is standard libs and second group are external imports